src/main/java/org/EmailCarbonFootprint/GmailAccountDetails.java

```diff
@@ -7,44 +7,45 @@
 public class GmailAccountDetails {
 
 	private String emailId;
-	private String appPassword;
-	
+	private String gmailAppPassword;
+
 	public GmailAccountDetails() {
 	}
-	
+
 	public GmailAccountDetails(String emailId, String appPassword) {
-		this.emailId =  emailId;
-		this.appPassword = appPassword;
+		this.emailId = emailId;
+		this.gmailAppPassword = appPassword;
 	}
+
 	public String getUserEmailId() {
 		return emailId;
 	}
+
 	public String getUserPassword() {
-		return appPassword;
+		return gmailAppPassword;
 	}
-	
-	public double getNumberOfInboxEmails(Store store) throws MessagingException {
-        Folder inbox = store.getFolder("Inbox");
-        inbox.open(Folder.READ_ONLY);
-        double numberOfInboxEmails = inbox.getMessageCount();
-        inbox.close(false);
-        return numberOfInboxEmails;
-    }
-
-    public double getNumberOfSpamEmails(Store store) throws MessagingException {
-        Folder spam = store.getFolder("[Gmail]/Spam");
-        spam.open(Folder.READ_ONLY);
-        double numberOfSpamEmails = spam.getMessageCount();
-        spam.close(false);
-        return numberOfSpamEmails;
-    }
-
-    public double getNumberOfSentEmails(Store store) throws MessagingException {
-        Folder sent = store.getFolder("[Gmail]/Sent Mail");
-        sent.open(Folder.READ_ONLY);
-        double numberOfSentEmails = sent.getMessageCount();
-        sent.close(false);
-        return numberOfSentEmails;
-    }
-}
 
+	public int getCountOfInboxEmails(Store emailDataStore) throws MessagingException {
+		Folder inboxCount = emailDataStore.getFolder("Inbox");
+		inboxCount.open(Folder.READ_ONLY);
+		int countOfInboxEmails = inboxCount.getMessageCount();
+		inboxCount.close(false);
+		return countOfInboxEmails;
+	}
+
+	public int getCountOfSpamEmails(Store emailDataStore) throws MessagingException {
+		Folder spamCount = emailDataStore.getFolder("[Gmail]/Spam");
```

Comment:
Similar comment for "spamCount" variable name and "sentCount" variable name as mentioned in line number 29